.chronus/changes/azhang_RemoveAzureResource-2024-6-3-12-45-15.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: internal
```

Comment:
doesn't feel like an internal change

Reply:
As we expect all ARM spec writers to extend from `Tracked/Proxy` or extreme cases `Resource`, spec writer should not use this nor the need to add `@extension("x-ms-azure-resource", true)` themselves. Hence I placed it in the `Private` namesapce. With the pending linting rule on `no reference Private namespace`, it is a pure internal matter, from outside (spec write et al) looking in, No?

Reply:
i mean this expose a new api, if that change is internal this means it shouildn't technically rquire a new release which isn't accurate here